src/lib/notifications/pushServiceFactory.ts

```diff
@@ -0,0 +1,14 @@
+import { PushService } from './pushServiceBase'
+import { Capacitor } from '@capacitor/core'
+import { AndroidPushService } from './pushServiceAndroid'
+import { WebPushService } from './pushServiceWeb'
+
+export function createPushService(): PushService {
+  if (Capacitor.isNativePlatform() && Capacitor.getPlatform() === 'android') {
```

Comment:
I think it's sufficient to check if platform is Android. No need in Capacitor.isNativePlatform()